test/bucket_test.cc

```diff
@@ -56,7 +56,6 @@ int compress_blob(hermes::api::TraitInput &input, hermes::api::Trait *trait) {
   size_t blob_size = bkt.Get(input.blob_name, blob, ctx);
   blob.resize(blob_size);
   bkt.Get(input.blob_name, blob, ctx);
-  bkt.Close(ctx);
```

Comment:
This bucket is not destroyed.

Reply:
There is the other bucket may that need to destroy in main #line 184

Reply:
I refactored the compression test into its own function so that it's easier to see which resources need to be freed. Thanks.